### PR TITLE
Add universal to the list of reloadables

### PR DESCRIPTION
**Pull request by bdraco**
Core PR : https://github.com/home-assistant/core/pull/39248

## Proposed change

Add universal to the list of reloadables

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [x] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/translations/en.json [hunk 1]
```diff
@@ -838,7 +838,8 @@
               "input_number": "Reload input numbers",
               "input_datetime": "Reload input date times",
               "input_select": "Reload input selects",
-              "template": "Reload template entities"
+              "template": "Reload template entities",
+              "universal": "Reload universal entities"
```
Comment (bramkragten) [inline src/translations/en.json:842]: Universal, is that just the media player? I really don't like the naming, and I guess it is confusing with the other "universal" integrations?

<https://www.home-assistant.io/integrations/#search/universal>

Comment (bramkragten) [reply src/translations/en.json:842]: We use "generic" for other platforms I think?

home-assistant.io/integrations/#search/generic

Comment (bdraco) [reply src/translations/en.json:842]: The naming choice is a bit odd ..... but it would be a breaking change to rename it. 🤷

Comment (bdraco) [reply src/translations/en.json:842]: Since its only `media_player` we could make it `Reload universal media player entities`

Comment (bdraco) [reply src/translations/en.json:842]: For reference https://github.com/home-assistant/core/issues/39033#issuecomment-676479193

Comment (bramkragten) [reply src/translations/en.json:842]: We should change our reload UI 😄 that's a long list...

Any idea what would happen if we called all reload services at the same time?

Comment (zsarnett) [reply src/translations/en.json:842]: I do like the idea that sean had for being able to click chips and reload many at once

Comment (zsarnett) [reply src/translations/en.json:842]: but if we are to do that lets get this PR in and then update the full UI in another

Comment (bdraco) [reply src/translations/en.json:842]: It should be relatively safe to reload a few at the same time.  We are basically doing that at startup anyways.

Comment (zsarnett) [reply src/translations/en.json:842]: Would it be best if there was some way to queue them?

Comment (bramkragten) [reply src/translations/en.json:842]:
```suggestion
              "universal": "Reload universal media player entities"
```

**Review by bramkragten — APPROVED** [review]

**Pull request merged**